Mage.Tests/src/test/java/org/mage/test/cards/abilities/activated/ChaosWandTest.java

```diff
@@ -0,0 +1,26 @@
+package org.mage.test.cards.abilities.activated;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.game.permanent.Permanent;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+public class ChaosWandTest  extends CardTestPlayerBase {
+
+    @Test
+    public void testChaosWant() {
+        addCard(Zone.BATTLEFIELD, playerA, "Chaos Wand");
+        addCard(Zone.BATTLEFIELD, playerA, "Plains", 4);
+        addCard(Zone.LIBRARY, playerB, "Blood Tithe");
+
+        activateAbility(1, PhaseStep.PRECOMBAT_MAIN, playerA, "{4}, {T}: ");
+        setChoice(playerA, "Yes");
+
+        setStopAt(1, PhaseStep.BEGIN_COMBAT);
+        execute();
```

Comment:
Same -- test commands wrong, but AI choosing helps. Try to use `setStrictChooseMode` all the time.

Improved test in 78a927943898ef1a5753e1fba557facaddd3e24c